File: beecrowd/1/1019/1019.cs
using System;

class URI {

    static void Main(string[] args) {
        int segundos = 0;
        int[] tempo = {60 * 60, 60, 1};
        segundos = int.Parse(Console.ReadLine());

        for (int i = 0; i < 3; i++) {
            int v = tempo[i];
            tempo[i] = segundos / v;
            segundos -= tempo[i] * v;
        }
        Console.WriteLine("{0}:{1}:{2}", tempo[0], tempo[1], tempo[2]);
    }
}

File: beecrowd/1/1019/1019.py
segundos = 0
tempo = [60 ** 2, 60, 1]

segundos = int(input())

for i in range(len(tempo)):
    v = tempo[i]
    tempo[i] = segundos // v
    segundos -= tempo[i] * v

print("%d:%d:%d" % (tempo[0], tempo[1], tempo[2]))
